examples/mp-metrics-counted/README_es.adoc

```diff
@@ -3,7 +3,7 @@
 :jbake-type: page
 :jbake-status: published
 
-Este es un ejemplo sobre cómo utilizar las métricas de microperfil en TomEE. El
+Este es un ejemplo sobre cómo utilizar las métricas de Microprofile en TomEE. El
```

Comment:
Can we use MicroProfile (just P in uppercase)

Reply:
Done!